rules/framework.bzl

```diff
@@ -1021,7 +1016,7 @@ that this target depends on.
         ),
         "_headermap_builder": attr.label(
             executable = True,
-            cfg = "host",
+            cfg = "exec",
```

Comment:
Was curious to know the implications of this change but apparently `host` was deprecated and effectively is pretty much the same as `exec` now. For other reviewers / future reference: https://bazel.build/extending/rules#configurations

Reply:
Yes, this is one that got handled by the autofix; most of the rest were semi-manual.